jdaviz/configs/imviz/tests/test_parser.py

```diff
@@ -16,6 +17,14 @@
 from jdaviz.configs.imviz.plugins.parsers import (
     parse_data, _validate_fits_image2d, _validate_bunit, _parse_image)
 
+try:
+    # check for version of roman_datamodels
+    import roman_datamodels
+    RDM_LT_0_14_2 = Version(roman_datamodels.__version__) < Version('0.14.2.dev')
+except ImportError:
+    # If roman_datamodels not installed, assume Roman-specific tests can be skipped
+    RDM_LT_0_14_2 = True
+
```

Comment:
`roman_datamodels > 0.14.1` will have tools for generating synthetic WFI-like data products on the fly. I've added a switch to allow tests to run when a sufficient version of roman_datamodels is installed.